docs/en_US/jailbreak/installing-kok3shi9.md

```diff
@@ -17,7 +17,7 @@ extra_contributors:
   - TheHacker894
 ---
 
-kok3shi9 is capable of jailbreaking all devices on 9.3.2 to 9.3.6, as well as 64-bit devices on 9.3 to 9.3.1
+kok3shi9 is capable of jailbreaking all devices on 9.3.2 to 9.3.6, as well as 64-bit devices on 9.3 to 9.3.5
```

Comment:
imo the original wording is better and is more consistent with the wording that other pages (such as the HomeDepot page) use. (Also avoids restating information in a weird way)

Reply:
Alright, I will revert it